packages/v4/patternfly-docs/content/accessibility/chip/chip.md

```diff
@@ -3,17 +3,58 @@ id: Chip
 section: components
 ---
 
-A **chip** is used to communicate a value or a set of attribute-value pairs within workflows that involve filtering a set of objects. A chip contains a text element, and usually also a button that allows the user to remove the chip from a selection. Read-only or draggable chips may not have a button.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should be able to navigate to the button on a chip, as long as the chip has a button, and remove it from a selection.
+## Accessibility
 
-**Screen reader users** should be able to navigate to the chip and have a screen reader read the chip’s text, as well as remove the chip from a selection if it has a button.
+To implement an accessible PatternFly **chip** component:
 
-The following attributes have been added for you or are customizable in PatternFly:
+- Ensure any close button within a chip can be navigated to and interacted with via keyboard and other assistive technologies such as a screen reader
+- Ensure the entire chip can receive focus, not just the chip's close button, when the chip content is truncated
+- Ensure any truncated text within a chip is displayed as a tooltip when the chip is hovered via mouse or has focus
+- Follow the [tooltip accessibility documentation](/components/tooltip/accessibility) for truncated chip content
```

Comment:
Same as note in chipgroup docs. 
It's worth leaving a note about the truncated chip being focusable in the testing section I think, but I like keeping these bullets at the top limited to the considerations the consumer is expected to make when implmeneting a Chip